doc/source/examples/unresolved-cfd-dem/single-particle-sedimentation/single-particle-sedimentation.rst

```diff
@@ -0,0 +1,414 @@
+Since we are calculating the void fraction using the particle insertion of the DEM simulation, we set the ``mode`` to ``dem``. For this, we need to read the dem files which we already wrote using check-pointing. We, therefore, set the ``read dem`` to ``true`` and specify the prefix of the dem files to be dem.
+We choose to use the quadrature centered method (`QCM <../../../theory/unresolved_cfd-dem/void-fraction.html>`_) to calculate the void fraction. For this, we specify the ``mode`` to be ``qcm``.
+
+We want the radius of our volume averaging sphere to be equal to the length of the element where the void fraction is being calculated. We do not want the volume of the sphere to be equal to the volume of the element. For this, we set the ``qcm sphere equal cell volume`` equals to ``false``. Then, we set the diameter of the QCM sphere to be twice the size of our particle's diameter. We also set the smoothing length equal to 10 times the particle diameter. Lastly, we choose the ``gauss-lobatto`` quadrature rule with 5 quadrature points. More details on these parameters are available on the `documentation on void fraction parameters <../../../theory/unresolved_cfd-dem/void-fraction.html>`_.
+
+.. code-block:: text
+
+    subsection void fraction
+      set mode                         = qcm
+      set qcm sphere equal cell volume = false
+      set qcm sphere diameter          = 0.005326
+      set read dem                     = true
+      set dem file name                = dem
+      set l2 smoothing length          = 0.02663
+      set quadrature rule              = gauss-lobatto
+      set n quadrature points          = 5
+    end
+
+
+CFD-DEM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We enable grad-div stabilization in order to improve local mass conservation. If we were using PCM and SPM void fraction schemes, the void fraction time derivative should be disabled as the time variation of the void fraction will lead to unstable simulations. The source of such instability is the first term of the continuity equation :math:`\rho_f \frac{\partial \varepsilon_f}{\partial t}`, which is stiff and unstable for the slightest temporal discontinuity of the void fraction and as :math:`\Delta t \to 0`. However, as we are using the QCM void fraction scheme, this term can be enabled. Usually, this term is neglected, however; disabling this term affects the results as we are no longer solving for the actual `Volume Averaged Navier-Stokes equations <../../../theory/multiphase/cfd_dem/unresolved_cfd-dem.html>`_. Therefore, we should not neglect this term based on numerical reasoning without any physical explanation.
+
+.. code-block:: text
+
+    subsection cfd-dem
+      set grad div                      = true
+      set void fraction time derivative = true
+      set drag force                    = true
+      set buoyancy force                = true
+      set shear force                   = true
+      set pressure force                = true
```

Comment:
If you want to compare with the analytical solution for your drag model, you should not pure shear force and pressure force.
```suggestion
      set shear force                   = false
      set pressure force              = false
```